sass/base/_compass-icons.scss

```diff
@@ -90,8 +101,8 @@
 .icon-jumbo-attachment-patch:before { content: '\e909'; } /* '' */
 .icon-jumbo-attachment-zip:before { content: '\e90a'; } /* '' */
 .icon-jumbo-attachment-code:before { content: '\e90b'; } /* '' */
+.icon-jumbo-attachment-image-broken:before { content: '\e90c'; } /* '' */
 .icon-account-multiple-outline:before { content: '\f00f'; } /* '' */
-.icon-account-outline:before { content: '\f013'; } /* '' */
```

Comment:
@michaelgamble is there a reason `icon-account-outline` is being removed?

Reply:
Actually looks like this one just moved further down ... git is not always the best at picking changes up with they are moved. See ln. 217 on the right side.